src/cpp/session/resources/themes/dreamweaver.rstheme

```diff
@@ -204,7 +204,7 @@
 .xtermBold { font-weight: bold; }
 .xtermBlur { filter: blur(1px); }
 .xtermUnderline { text-decoration: underline; }
-.xtermHyperlink { text-decoration: underline; cursor: pointer; text-decoration-style: dotted; color: #75507b; }
```

Comment:
Are these changes intentional?

Reply:
I think it's a merge hiccup: 

 - I merged from here from one of your review comment
 - but I also had changed locally 

so things got messy when I `git pull` locally or something like that. But these changes are in main already, from a recent pr: https://github.com/rstudio/rstudio/pull/11147

so I think it should be good to squash.